scripts/zones/Pashhow_Marshlands/mobs/BoWho_Warmonger.lua

```diff
@@ -3,16 +3,67 @@
 --   NM: Bo'Who Warmonger
 -----------------------------------
 require("scripts/globals/regimes")
+mixins = {require("scripts/mixins/job_special")}
+local ID = require("scripts/zones/Pashhow_Marshlands/IDs")
 -----------------------------------
 local entity = {}
 
+-- TODO: Implement better pathing systems for guards to follow master
+
+entity.onMobSpawn = function(mob)
+    -- Takes half damage from all attacks
+    mob:addMod(xi.mod.UDMGPHYS,-5000)
```

Comment:
[one mod](https://github.com/LandSandBoat/server/blob/base/scripts/globals/status.lua#L1066) to rule them all and [in the dmg halve them](https://github.com/LandSandBoat/server/search?q=mod%3A%3Admg)

Reply:
see also: all comments I left regarding meteo mauler on yoru other pr, his adds, ids.lua entries, etc etc.

All these beastmen lair guys are built similarly.

Reply:
everything else on pr looks good to go now, as far as my almost-2am-brain can tell :+1:

Reply:
Got that adjusted for this guy now.

Reply:
I thought you swapped out the mod already here. `xi.mod.DMG` covers all 4. the `U` version just mean uncapped btw, and 50% doesn't exceed that cap (mobs don't even cap the same way players do anyway).